aiven/client/cli.py

```diff
@@ -4,17 +4,19 @@
 # See the file `LICENSE` for details.
 from . import argx, client
 from aiven.client import envdefault
-from aiven.client.cliarg import arg
+from aiven.client.cliarg import arg  # type: ignore
```

Comment:
No, it's because `mypy` complains that `Module "aiven.client.cliarg" has no attribute "arg"`. This is perhaps because of the way we structured this lib.

Reply:
Ah, this is because it's imported from `.argx` there. I think it was already discussed in another PR. But maybe do `arg: Type` in `cliarg` will help resolve it?